.vscode/scripts/run-hvacservice.sh

```diff
@@ -13,46 +14,48 @@
 # SPDX-License-Identifier: Apache-2.0
 
 echo "#######################################################"
-echo "### Running HVAC Service from VAL                   ###"
+echo "### Running HVAC Service                            ###"
 echo "#######################################################"
 
 ROOT_DIRECTORY=$( realpath "$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )/../.." )
 source $ROOT_DIRECTORY/.vscode/scripts/exec-check.sh "$@" $(basename $BASH_SOURCE .sh)
 
-SERVICE_VERSION=$(cat $ROOT_DIRECTORY/prerequisite_settings.json | jq .hvacservice.version | tr -d '"')
-SERVICE_PORT='50052'
-SERVICE_GRPC_PORT='52005'
-HVACSERVICE_DAPR_APP_ID='hvacservice'
+HVACSERVICE_IMAGE=$(cat $ROOT_DIRECTORY/prerequisite_settings.json | jq .hvacservice.image | tr -d '"')
+HVACSERVICE_TAG=$(cat $ROOT_DIRECTORY/prerequisite_settings.json | jq .hvacservice.version | tr -d '"')
```

Comment:
same, here, should we use app-manifest?

Reply:
See comments to prerequisite_settings.json